src/cli/Microsoft.DotNet.UpgradeAssistant.Cli/appsettings.json

```diff
@@ -10,7 +10,7 @@
     "PackageMapPath": "PackageMaps"
   },
   "TemplateInserter": {
-    "TemplatePath": "Templates"
+    "TemplateConfigFiles": [ "Templates\\CSharpWebAppTemplates\\TemplateConfig.json" ]
```

Comment:
Super nit-pick - we should split this across a few lines to match how other arrays are formatted in this file.